packages/babel-runtime-corejs2/package.json

```diff
@@ -913,8 +913,8 @@
     "./package.json": "./package.json",
     "./regenerator": "./regenerator/index.js",
     "./regenerator/*.js": "./regenerator/*.js",
-    "./regenerator/": "./regenerator/",
```

Comment:
The trailing slash is needed for compatibility with a few node versions that didn't support subpath patterns. The pattern with `*` is defined in the line above (`"./regenerator/*.js": "./regenerator/*.js"`).

Reply:
ok, thanks!